mage_ai/data_preparation/executors/block_executor.py

```diff
@@ -27,11 +27,14 @@ def execute(
         update_status: bool = False,
         on_complete: Callable[[str], None] = None,
         on_failure: Callable[[str], None] = None,
+        on_start: Callable[[str], None] = None,
```

Comment:
@dy46 this is not used anymore, right?

Reply:
It's not, but I chose to keep it because I could see it being used in the future.